flag_engine/features/schemas.py

```diff
@@ -35,7 +37,7 @@ class Meta:
 
 
 class BaseFeatureStateSchema(Schema):
```

Comment:
We need to make sure that we add `unknown = EXCLUDE` to the `Meta` class of this schema now.